plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3/TrinoS3FileSystem.java

```diff
@@ -295,10 +295,12 @@ public void initialize(URI uri, Configuration conf)
     }
 
     @Override
+    @SuppressWarnings("UnnecessaryMethodReference")
     public void close()
             throws IOException
     {
         try (Closer closer = Closer.create()) {
+            // https://github.com/google/error-prone/issues/2102
```

Comment:
i'd put this comment on `@SuppressWarnings` line, because it matters there, not here